src/components/Header/HeaderNav/HeaderNav.js

```diff
@@ -6,11 +6,11 @@ const HeaderNav = ({ nav }) => {
       <div className="menu-main--container">
         <ul className="menu-main">
           {nav.map((item, index) => (
-            <li className="submenu-item" key={index}>
+            <li className={item.subNav.length ? 'submenu-item' : ''} key={index}>
               <a className={item.isSelected ? 'active' : ''} href={item.url}>
                 {item.title}
               </a>
-              {item.subNav ? (
+              {item.subNav.length ? (
```

Comment:
Are you sure this does not break? Menus without items have no subNav property.

Reply:
![image](https://user-images.githubusercontent.com/2439363/57634444-b33c2000-757b-11e9-9a0c-6c3f21c715ef.png)
It works but it gives console errors...

Reply:
Working with @damianmuti,  I have checked the mapper and it seems that `item.subNav` is never null, only empty array. 

How did you tested it? Did you modify headerData.json?

Reply:
Not at all. I've seen this issue seems to be from UI library, because the error is in JQuery, this may be from before this code.

Reply:
Me parece que ese error ya existía, puede ser? en ese caso me gustaría dejarlo fuera del scope de este PR. ¿Me podrás confirmar?

Reply:
Si es de la librería del jquery y no de lo que se toco puntualmente en el pr asociado, se introdujo aca https://cdn.fromdoppler.com/doppler-ui-library/build247/header.html en el anterior no https://cdn.fromdoppler.com/doppler-ui-library/build246/header.html